chia/server/ws_connection.py

```diff
@@ -272,11 +283,11 @@ async def invoke(*args, **kwargs):
             msg = Message(uint8(getattr(ProtocolMessageTypes, attr_name).value), None, args[0])
             request_start_t = time.time()
             result = await self.create_request(msg, timeout)
-            self.log.debug(
+            self.log.info(
                 f"Time for request {attr_name}: {self.get_peer_logging()} = {time.time() - request_start_t}, "
                 f"None? {result is None}"
             )
-            if result is not None:
+            if result is not None and result.data != b"":
```

Comment:
if `result is not None and result.data != b""` is important protocol semantics, you might want to stick it in a function, just to have the name as documentation.

Reply:
good idea

Reply:
i changed this so it will be clearer, the check for != b"" just to make sure we don't try to parse the empty bytes